src/platform/packages/shared/kbn-unified-tabs/src/types.ts

```diff
@@ -19,6 +19,10 @@ export interface TabItem {
   customMenuButton?: React.JSX.Element;
 }
 
+export type RecentlyClosedTabItem = TabItem & {
+  closedAt: number;
```

Comment:
This makes the dependency on `closedAt` more explicit. Previously, it was a shadow property that was being used but not disclosed in the unified tabs package interfaces.